nifi-nar-bundles/nifi-box-bundle/nifi-box-processors/src/main/resources/docs/org.apache.nifi.processors.box.ListBoxFiles/additionalDetails.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>ListBoxFiles</title>
+
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+</head>
+<body>
+<p>This processor requires a pre-configured App under the User owning the Folder from which files are being listed.</p>
+<p>The App should have the following configuration:</p>
+<ul>
+    <li>Authentication method should be 'OAuth 2.0 with JSON Web Tokens (Server Authentication)'</li>
```

Comment:
When a new app is created the authentication method can be selected as "Server Authentication (with JWT)".
Later on it is displayed in App details as you mentioned  ('OAuth 2.0 with JSON Web Tokens (Server Authentication)'). 
Maybe it's worth mentioning the other format of authentication method name as well to avoid confusion.